src/main/typescript/components/form/parts/fileUpload/overview/FilesOverview.tsx

```diff
@@ -18,78 +18,64 @@ import { Component } from "react"
 import FilesTableHead from "./FilesTableHead"
 import FilesTableRow from "./FilesTableRow"
 import "../../../../../../resources/css/filesOverviewTable.css"
-import axios from "axios"
-import { listFiles } from "../../../../../constants/serverRoutes"
 import { DepositId } from "../../../../../model/Deposits"
-
-// <temp>
-export interface FileInfo {
-    filename: string
-    dirpath: string
-    sha1sum: string
-}
-// </temp>
+import { FileOverviewState } from "../../../../../model/FileInfo"
+import { PromiseAction } from "../../../../../lib/redux"
+import { connect } from "react-redux"
+import { cleanFiles, fetchFiles } from "../../../../../actions/fileOverviewActions"
+import { AppState } from "../../../../../model/AppState"
+import { Action } from "redux"
 
 interface FilesOverviewProps {
     depositId: DepositId
-}
+    files: FileOverviewState
 
-interface FilesOverviewState {
-    files: { [filepath: string]: FileInfo }
-    fetchingFiles: boolean
-    fetchedFiles: boolean
+    cleanFiles: () => Action
+    fetchFiles: (depositId: DepositId, dirPath: string) => PromiseAction<void>
 }
 
-class FilesOverview extends Component<FilesOverviewProps, FilesOverviewState> {
+class FilesOverview extends Component<FilesOverviewProps> {
     constructor(props: FilesOverviewProps) {
         super(props)
-        this.state = {
-            files: {},
-            fetchingFiles: false,
-            fetchedFiles: false,
-        }
     }
 
     async componentDidMount() {
-        this.setState(prevState => ({ ...prevState, fetchingFiles: true }))
-
-        const url = await listFiles(this.props.depositId)
-        const response = await axios.get(url)
-
-        const data = response.data
-            .reduce((collected: { [filepath: string]: FileInfo }, info: any) => ({
-                ...collected,
-                [info.dirpath + info.filename]: {
-                    filename: info.filename,
-                    dirpath: info.dirpath,
-                    sha1sum: info.sha1sum,
-                },
-            }), {})
+        this.props.fetchFiles(this.props.depositId, "")
```

Comment:
This is the only call to the `fetchFiles` action. It uses an empty `string` for the second argument. Do you intent to use this action somewhere else where this is not the case? If not, please remove the second argument.